product_docs/docs/epas/15/working_with_oracle_data/06_dblink_ora/index.mdx

```diff
@@ -12,3 +12,5 @@ redirects:
 
 
 dblink_ora enables you to issue arbitrary queries to a remote Oracle server. It provides an OCI-based database link that allows you to `SELECT`, `INSERT`, `UPDATE`, or `DELETE` data stored on an Oracle system from EDB Postgres Advanced Server.
+
+The following topics describe how to use the dblink_ora feature.
```

Comment:
In a conversation I had with Betsy yesterday, she said to remove these types of intro sentences.